app/frontends/jobs.ts

```diff
@@ -221,3 +221,42 @@ export async function cancelJob(
     }
   }
 }
+
+/**
+ * Express.js handler that resumes a single job `(POST /jobs/{jobID})`. A user can resume their own
```

Comment:
Looks like the route is /jobs/:jobID/resume

Reply:
Good catch. Copied from the `cancelJob` function above. Fixed here and there was well.